### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by geoand**
Small changes to grammar and wording

### CODE REVIEW

**Comment by yole** [comment]
Please don't remove the 'public' modifier from the standard library methods; it is left there intentionally.

**Comment by geoand** [comment]
Sorry about that, the IDE seems to have done that automatically!
I'll fix it immediately

**Comment by geoand** [comment]
If you decide to merge this, let me know if you want me to the rebase the branch so that only one (or possibly two) commits will appear in the log instead of three (where the third one corrects the erroneous removal of the public modifiers)

File: libraries/stdlib/src/kotlin/collections/MapWithDefault.kt [hunk 1]
```diff
@@ -24,11 +24,11 @@ internal fun <K, V> Map<K, V>.getOrImplicitDefault(key: K): V {
...
  * Returns a wrapper of this read-only map, having the implicit default value provided with the specified function [defaultValue].
  * This implicit default value is used when properties are delegated to the returned map,
- * and that map doesn't contain value for the key specified.
+ * and that map doesn't contain a value for the key specified.
  *
- * When this map already have an implicit default value provided with a former call to [withDefault], it is being replaced by this call.
+ * When this map already has an implicit default value provided with a former call to [withDefault], it is being replaced by this call.
  */
-public fun <K, V> Map<K, V>.withDefault(defaultValue: (key: K) -> V): Map<K, V> =
+fun <K, V> Map<K, V>.withDefault(defaultValue: (key: K) -> V): Map<K, V> =
```
Comment (ilya-g) [inline libraries/stdlib/src/kotlin/collections/MapWithDefault.kt]: Missing `public` modifier.

**Comment by ilya-g** [comment]
@geoand Yes,  please do a rebase and squash the third commit with the first one.

**Comment by geoand** [comment]
@ilya-g All set :)

File: ReadMe.md [hunk 2]
```diff
@@ -117,7 +117,7 @@ and find classes or functions which are not documented very well and submit a pa
 In particular it'd be great if all functions included a nice example of how to use it such as for the
 <a href="http://kotlinlang.org/api/latest/jvm/stdlib/kotlin/hash-map-of.html">hashMapOf()</a> function.
 This is implemented using the <a href=https://github.com/JetBrains/kotlin/blob/master/libraries/stdlib/src/kotlin/collections/Maps.kt#L53">@sample</a>
-macro to include code from a test function. This serves as a double win; the API gets better documented with nice examples to help new users and the code gets more test coverage.
+macro to include code from a test function. The benefits of this approach are twofold; First, the API's documentations is improved via beneficial examples that help new users and second, the code coverage is increased.
```
Comment (ilya-g) [inline ReadMe.md]: > documentations is

Not sure if it's consistent: either "documentations are" or "documentation is"

**Comment by geoand** [comment]
"Documentation" in this sense is a single entity, and therefore should be "is"

**Comment by ilya-g** [comment]
Ok, I've fixed minor issues by myself and merged manually, thanks.

**Comment by geoand** [comment]
Ok, thanks!

**Pull request closed without merging**